statistics/histogram_test.go

```diff
@@ -346,21 +346,21 @@ func (s *testStatisticsSuite) TestMergePartitionLevelHist(c *C) {
 					upper:  5,
 					count:  10,
 					repeat: 1,
-					ndv:    3,
+					ndv:    0,
 				},
 				{
 					lower:  5,
 					upper:  12,
 					count:  22,
 					repeat: 3,
-					ndv:    5,
+					ndv:    0,
 				},
 				{
 					lower:  12,
 					upper:  18,
 					count:  33,
 					repeat: 5,
-					ndv:    5,
+					ndv:    0,
```

Comment:
Could you please modify `MergePartitionHist2GlobalHist(sc, hists, poped, t.expBucketNumber, false)` to `MergePartitionHist2GlobalHist(sc, hists, poped, t.expBucketNumber, true)` instead of changing `ndv`?

Reply:
OK, I'll modify it soon.